core/src/main/java/org/openstack4j/api/storage/BlockVolumeService.java

```diff
@@ -41,6 +42,27 @@
      */
     VolumeType createVolumeType(VolumeType volumeType);
 
+    /**
+     * Creates a new encryption type with the specified instance for the specified volume type
+     *
+     * @param encryptionType
+     *            the EncryptionType for create
+     * @param volumeTypeId
+     *            the volume type identifier
+     * @return the created encryption type
+     */
+    EncryptionType createEncryptionType(EncryptionType encryptionType, String volumeTypeId);
```

Comment:
Given the encryption type is a feature of a volume type, I suggest reversing the order of the parameters.

Reply:
Done